src/realm/util/file.cpp

```diff
@@ -1010,6 +1012,10 @@ bool File::try_remove(const std::string& path)
 
 void File::move(const std::string& old_path, const std::string& new_path)
 {
+#ifdef _WIN32
+    // Can't rename to existing file on Windows
+    try_remove(new_path);

```

Comment:
Should probably bubble up errors other than "file not found"

Reply:
(a FIXME about it would be fine for now I think)

Reply:
I don't understand. try_remove() will not, and should not, report any errors.

Reply:
(simon and I just discussed it on slack and it's ok)